x-pack/legacy/plugins/alerting/server/alerts_client.test.ts

```diff
@@ -418,16 +418,9 @@ describe('create()', () => {
     await expect(alertsClient.create({ data })).rejects.toThrowErrorMatchingInlineSnapshot(
       `"Task manager error"`
     );
-    expect(alertsClientParams.log).toHaveBeenCalledTimes(1);
-    expect(alertsClientParams.log.mock.calls[0]).toMatchInlineSnapshot(`
-                                                                                                                  Array [
-                                                                                                                    Array [
-                                                                                                                      "alerting",
-                                                                                                                      "error",
-                                                                                                                    ],
-                                                                                                                    "Failed to cleanup alert \\"1\\" after scheduling task failed. Error: Saved object delete error",
-                                                                                                                  ]
-                                                                            `);
```

Comment:
holy cow!  How does prettier get this so wrong??? :-)

Reply:
I figured just doing `expect(alertsClientParams.logger.error).toHaveBeenCalledWith(...)` is prettier now 😄